pkg/securitycontextconstraints/sccadmission/admission.go

```diff
@@ -224,6 +225,9 @@ func (c *constraint) computeSecurityContext(ctx context.Context, a admission.Att
 
 	providers, errs := sccmatching.CreateProvidersFromConstraints(a.GetNamespace(), constraints, c.client)
 	logProviders(pod, providers, errs)
+	if len(errs) > 0 {
+		return nil, "", nil, kutilerrors.NewAggregate(errs)
```

Comment:
the pod might still pass the admission, right?

Reply:
I meant if we did not return the error. Maybe we shouldn't if admission might still pass? I suspect that might be why the unit tests are failing.

Reply:
@stlaz Yes, you're right, most likely the original design rationale was:

Given a set of SCC constraints (A, B, C) (in rank order) if one of them fails to be converted to provisioners, i.e. B, C then currently this will still go through admission.

The question though is if this is actually correct from a logical perspective. B, and C have lower rank than A so might be more permissive which would be even more fatal. Currently, we fortunately forbid admission because the underlying service account doesn't have permissions for B, and C.

But if the underlying service account _would_ have permissions we would admit lower ranked SCC permissions just because of i.e. a network blip.